frontend/src/components/table/schedule/sections/shifts-section/shifts-section.tsx

```diff
@@ -0,0 +1,62 @@
+import { Button } from "@material-ui/core";
+import React from "react";
+import { DataRowHelper } from "../../../../../helpers/row.helper";
+import { StringHelper } from "../../../../../helpers/string.helper";
+import { DataRow } from "../../../../../logic/real-schedule-logic/data-row";
+import { ShiftsInfoLogic } from "../../../../../logic/real-schedule-logic/shifts-info.logic";
+import { WorkerTypeHelper } from "../../../../../state/models/schedule-data/employee-info.model";
+import { Shift } from "../../../../../state/models/schedule-data/shift-info.model";
+import { ShiftCellComponent } from "../../schedule-parts/shift-cell.component";
+import { BaseSectionComponent } from "../base-section/base-section.component";
+import "./shifts-section.css";
+import { ShiftsSectionOptions } from "./shifts-section.options";
+
+const sh = StringHelper;
+
+export function ShiftsSectionComponent({
+  workerType,
+  data = [],
+  metaDataLogic,
+  onSectionUpdated,
+}: ShiftsSectionOptions) {
+  const logic = new ShiftsInfoLogic(DataRowHelper.dataRowsAsValueDict<Shift>(data, true));
+
+  function addDataRow(newRow: DataRow) {
```

Comment:
@pectom 
Jak będziesz implementować przeciąganie wierszy, to po dodaniu jakiejś funkcji która reaguje na opuszczenie wiersza do sekcji, mógłbyś skorzystać z `addDataRow`